src/current/v23.2/begin-transaction.md

```diff
@@ -78,7 +77,18 @@ Without modifying the `BEGIN` statement, the transaction uses `SERIALIZABLE` iso
 > COMMIT;
 ~~~
 
-{{site.data.alerts.callout_danger}}This example assumes you're using <a href="{% link {{ page.version.version }}/transaction-retry-error-reference.md %}#client-side-retry-handling">client-side retry handling</a>.{{site.data.alerts.end}}
+{{site.data.alerts.callout_danger}}
+This example assumes you're using <a href="{% link {{ page.version.version }}/transaction-retry-error-reference.md %}#client-side-retry-handling">client-side retry handling</a>.
+{{site.data.alerts.end}}
+
+#### Change isolation level
+
+You can set the transaction isolation level to [`SERIALIZABLE`]({% link {{ page.version.version }}/demo-serializable.md %}) or [`READ COMMITTED`]({% link {{ page.version.version }}/read-committed.md %}). If not specified, transactions use the value of the current session's [`default_transaction_isolation`]({% link {{ page.version.version }}/session-variables.md %}#default-transaction-isolation) variable.
```

Comment:
Would it be worth adding a comment here about `READ COMMITTED` actually being `SERIALIZABLE` if `sql.txn.read_committed_isolation.enabled` is false?

Reply:
Good point -- I added this to the section in the Read Committed doc where that cluster setting is described (and linked this section directly to that section in the Read Committed doc), as I think it's clearest and most upfront there.